map-of-the-web/motw/motw/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for motw project.
+
+Generated by 'django-admin startproject' using Django 2.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'l$6!exjhsmmqhtx1q8=v(c^*&0i4$unkr@p%9hewp#@w89)(xo'
```

Comment:
Heads up: this is a secret and should NOT be committed to a repository!  Since it has been, you should generate a new secret.  There don't seem to be other places in this file with private secrets but do a review as well just in case.

In general be very careful about committing code that might have secret information in it!  I appreciate that this is just for development right now, but on principle it is good to treat all secrets as secret just in case.

Of course, the secrets have to live somewhere. I found [this article](http://fearofcode.github.io/blog/2013/01/15/how-to-scrub-sensitive-information-from-django-settings-dot-py-files/) that gives some pathways to scrubbing your django settings.  (I googled "django settings and github" to find it)